.changeset/hip-chairs-double.md

```diff
@@ -0,0 +1,20 @@
+---
+'@backstage/plugin-tech-radar': minor
```

Comment:
Think that this should also be a `patch` change too, as the `tech-radar` plugin is `0.x.x`. So breaking changes would be `minor` in this case.